sdk/containerregistry/container-registry/package.json

```diff
@@ -15,11 +15,7 @@
     "constantPaths": [
       {
         "path": "src/generated/generatedClientContext.ts",
-        "prefix": "packageVersion"
-      },
-      {
-        "path": "src/constants.ts",
-        "prefix": "SDK_VERSION"
+        "prefix": "const packageDetails"
```

Comment:
@jeremymeng do you remember why having the `const` part here?

Reply:
Because this metadata is named "prefix" and the line 

`    const packageDetails = `azsdk-js-container-registry/1.0.0-beta.4`;`

contains `const packageDetails` at the beginning.  However it doesn't have to be here, if `packageDetails` alone also works when the bot is bumping versions.